internal/test/fake_source.go

```diff
@@ -0,0 +1,26 @@
+package test
```

Comment:
Can you remove this file, the `Faker` structure should be enough to fake the seed

Reply:
I understand that I have to use `Faker.NewWithSource` but how will I be able to control the returned number from the `rand.Source` without a dummy implementation like this?

I apologize if I'm not following. I think might be missing something from the implementation of `rand.Source`so if you could provide a small sample on how to control the returned numbers it would be great.